example/example.js

```diff
@@ -0,0 +1,17 @@
+import Geobatch from '../src/index';
+
+const accessor = input => input.address,
+  input = [
+    {
+      address: 'Hamburg'
+    },
+    {
+      address: 'Berlin'
+    }
+  ],
+  gb = new Geobatch({}, accessor),
+  resultStream = gb.geocode(input);
```

Comment:
Use consistent naming. In the Readme it is always named `GeoBatch` or `geoBatch`.